### ISSUE TITLE
Vet Center CAP -> Lighthouse push

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#5375]
Blocked by #8627

Should this be a push to lighthouse or a pull from lighthouse? What is their preference?

**Comment by kevwalsh** [comment]
closing this in favor of #8683, now that we are an integrated team.
